app/main.cpp

```diff
@@ -99,50 +99,76 @@ static void setEnvironmentQgisPrefixPath() {
   qDebug() << "QGIS_PREFIX_PATH: " << ::getenv("QGIS_PREFIX_PATH");
 }
 
+static bool cpDir(const QString &srcPath, const QString &dstPath)
+{
+    QDir parentDstDir(QFileInfo(dstPath).path());
+    QFile::setPermissions(dstPath, QFile::ReadUser|QFile::WriteUser|QFile::ReadOwner|QFile::WriteOwner);
```

Comment:
Before the permissions were getting set only for files, now they are only being set for directories... why?

Reply:
Seems like if files (demo projects or resources) are copied to destination in internal/external memory, its not necessary to set permissions. Its different with system folders.